run

```diff
@@ -75,12 +76,12 @@ if [ "$LOCAL" = true ]; then
     fi
 fi
 
-yarn install --frozen-lockfile --silent >/dev/null 2>&1 || {
+yarn install --immutable >/dev/null 2>&1 || {
 echo "Dependency install failed (root)" >&2; exit 1; }
 pushd cli >/dev/null
-yarn install --frozen-lockfile --silent >/dev/null 2>&1 || {
+yarn install --immutable >/dev/null 2>&1 || {
 echo "Dependency install failed (cli)" >&2; popd >/dev/null; exit 1; }
 popd >/dev/null
 
 # run the CLI
-yarn --silent tsx cli/run.ts "${ARGS[@]-}"
+./cli/run.ts "${ARGS[@]-}"
```

Comment:
This is excellent, thanks for doing this and fixing all the files in core (in a previous PR) so that we can take advantage of this!